oaknational/Oak-Web-Application
# CTC-59: browse data for media clips page

## Description

Music year: [2017](https://www.youtube.com/watch?v=_NSuIYwBxu4)

Teacher media clip pages now render their data from a query requests rather than hardcoded data on the page

- Creates `lessonMediaClips` directory with query, test files, schemas and fixture data
- Creates construct functions for both canonical and canonical lesson media clips 
- Adds these queries to the media pages so they render lesson data via the `getStaticProps` method


## Issue(s)

Fixes #CTC-59

### Acceptance Criteria

- [ ]  A stub query is created for media clips page
- [ ]  browse data returned from getStaticProps
- [ ]  breadcrumbs implemented
- [ ]  handles canonical and browse versions

## How to test

1. Go to https://deploy-preview-3013--oak-web-application.netlify.thenational.academy/teachers/programmes/chemistry-secondary-ks4-foundation-edexcel/units/atomic-structure-and-the-periodic-table/lessons/atomic-structure-negligible-electron-mass/media
https://deploy-preview-3013--oak-web-application.netlify.thenational.academy/teachers/lessons/atomic-structure-negligible-electron-mass/media
 - or append /media to any lesson page
2. You should see a media page with a working breadcrumb component

## Screenshots

How it should now look:
{screenshots}
#### Canonical Lesson
<img width="1337" alt="Screenshot 2024-11-22 at 16 46 29" src="https://github.com/user-attachments/assets/02e7a917-b896-487a-b6d6-02ce0735e50b">

#### Non-canonical Lesson
<img width="1339" alt="Screenshot 2024-11-22 at 16 46 45" src="https://github.com/user-attachments/assets/5eebff77-4f25-47d2-a20d-d3230f44164a">

## Checklist
